apps/storybook/.eslintrc.cjs

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  root: true,
+  env: { browser: true, es2020: true },
+  extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended', 'plugin:react-hooks/recommended', 'plugin:storybook/recommended'],
```

Comment:
I think we just need to add `'plugin:storybook/recommended'` to our eslint-config because the linter will have to run the code against components in other packages

Reply:
it should only be installed where we will have a .stories component. For example, in the future packages/ui lib